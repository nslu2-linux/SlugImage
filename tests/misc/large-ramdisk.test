#!/bin/sh

# This test is expected to fail with a fatal error.
../slugimage -p -o /dev/null -k files/1MiB-16 -r Ramdisk:files/6656KiB 2> $1

