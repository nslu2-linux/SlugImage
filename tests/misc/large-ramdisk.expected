Ran out of flash space in <Ramdisk> - 0x00010 bytes too large.
